src/com/pentalog/pentastagiu/homework2/problema2/Product.java

```diff
@@ -0,0 +1,38 @@
+package com.pentalog.pentastagiu.homework2.problema2;
+
+public class Product {
+
+    private static int id = 0;
```

Comment:
aceeasi idee cu id-ul ca si in clasa Customer + inca o observatie: e ok ca e declarat private, doar oare e ok ca ai adaugat setter pentru el? cu setterul poate sa vina oricine si sa iti reseteze counterul la 0 dupa ce tu ai creat deja 10 obiecte si logica ta de a crea niste identificatori unici o sa fie data peste cap